distributions/openhab-online/readme.md

```diff
@@ -0,0 +1,37 @@
+openHAB online distro
+---------------------
+
+
+The online distro doesn't contain many features.
+You'll need to install those manually for now.
+
+
+1. start openhab
+	start.sh
+
+2. view available features
+	feature:list
+
+3. install a feature e.g. install rest docs
+	feature:install openhab-misc-restdocs
```

Comment:
> I always considered minimal runtime as the smallest possible openHAB 'master' instance.

Well, this is then more or less what the openhab-runtime feature (https://github.com/kaikreuzer/openhab-core/blob/master/features/karaf/src/main/feature/feature.xml#L24) is meant for.
What I call "minimal" rather corresponds to the openhab-runtime-base feature (https://github.com/kaikreuzer/openhab-core/blob/master/features/karaf/src/main/feature/feature.xml#L14).

>  then filling out a config file will be a challenge too

Then openHAB isn't the right tool for that guy, because he will also have to go on and create item, sitemap and rule files, not to forget about the binding config files :-)
But yes: I think the config file could be delivered with "demo", so that this is automatically installed, if the user does not have any other configuration (e.g. because he only updated his instance and has kept his old "conf" folder).

> I'm not sure what you mean with automatically constructed.

Well, having a "conf" folder where all the individual setup is described - through item files, binding configuration files, etc., but hence also through the configuration of what addons have to be available (at the moment people copy their "addons" folder with the jars over for this purpose - it is more or less a replacement for the "addons" folder.
Think also about providing a preconfigured docker image for the RaspPi or other. There are many cases where you simply want to say, which addons should be part of your runtime.